test/JDBC/upgrade/15_7/schedule

```diff
@@ -516,4 +516,10 @@ babel_test_int8_numeric_oper
 babel_test_int2_numeric_oper
 BABEL_3571
 test_like_for_AI
+test_conv_string_to_date-before-16_3
+test_conv_string_to_datetime
+test_conv_string_to_datetime2-before-16_3
+test_conv_string_to_datetimeoffset-before-16_3
+test_conv_string_to_smalldatetime
+test_conv_string_to_time-before-16_3
```

Comment:
We are going to backpatch this to 15 and 14 as well so then ideally we shouldn't be using `-before` suffix in 15_7 and 14_12 schedule files but it will fail until we actually merge this change to those versions. So, let's make sure to have PRs for the previous versions merge them from 14 to 16 order.

Reply:
yes, we will merge this PR, once the PR's from previous version's gets merged